elide-datastore/elide-datastore-multiplex/src/main/java/com/yahoo/elide/datastores/multiplex/MultiplexTransaction.java

```diff
@@ -140,6 +141,18 @@ protected DataStoreTransaction getTransaction(Class<?> cls) {
         return transaction;
     }
 
+    protected DataStoreTransaction getRelationTransaction(Object object, String relationName, RequestScope scope) {
+        com.yahoo.elide.core.RequestScope requestScope;
+        try {
```

Comment:
The only reason we do this cast is to get the dictionary.  However, it looks like the dictionary can be fetched from multiplexManager.getDictionary().